management-model/src/main/java/org/terracotta/management/cluster/Server.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terracotta.management.cluster;
+
+import java.io.Serializable;
+
+/**
+ * @author Mathieu Carbou
+ */
+public final class Server extends AbstractManageableContainer<Stripe, Server> implements Serializable {
+
+  public static final String KEY = "serverName";
+
+  private String hostName; // matches xml config
+  private String bindAddress; // matches xml config
+  private int bindPort; // matches xml config
+  private int groupPort; // matches xml config
+  private ServerState state = ServerState.UNKNOWN; // taken at runtime from servers
+  private String version; // taken at runtime from servers
+  private long startTime; // taken at runtime from servers
```

Comment:
you mean the server class ? Yes it is! Otherwise we could not have any manageable on the servers.